src/lambda/expression.rs

```diff
@@ -36,46 +38,22 @@ pub enum Context {
   Path(Vec<String>),
 }
 
-#[derive(Clone)]
+#[derive(Clone, Debug)]
 pub enum Unsafe {
-  Http(
-    RequestTemplate,
-    Option<GroupBy>,
-    Option<Arc<DataLoader<HttpDataLoader, NoCache>>>,
-  ),
+  Http {
+    req_template: RequestTemplate,
+    group_by: Option<GroupBy>,
+    dl_id: Option<usize>,
```

Comment:
actually usize is too much generic type. To do it more rusty way we may consider using [newtype](https://matklad.github.io/2018/06/04/newtype-index-pattern.html) pattern to have specific id type for resolving the dl. This way we won't confuse what does usize means and how to properly acquire the dataloader.

Reply:
@meskill added a `struct DataLoaderId(usize)` to use `dl_id`. Could you please check if the current changes are the way you suggested or if there's something you would like me to change?

Reply:
@meskill added the type parameter just as you suggested